src/main/java/uk/gov/justice/digital/delius/service/ConvictionService.java

```diff
@@ -338,6 +342,55 @@ public Custody addOrReplaceOrDeleteCustodyKeyDates(Long offenderId, Long convict
                 .getCustody());
     }
 
+    public Optional<ProbationStatusDetail> probationStatusFor(String crn) {
+        return offenderRepository.findByCrn(crn)
+            .filter(offender -> offender.getSoftDeleted() == 0L)
+            .map((offender) -> new ProbationStatusDetail(
+                probationStatusOf(offender),
+                previouslyKnownTerminationDateOf(offender),
+                inBreachOf(offender),
+                preSentenceActivityOf(offender)));
+    }
+
+    private ProbationStatus probationStatusOf(Offender offender) {
+        if (offender.getCurrentDisposal() == 1)
```

Comment:
Just in terms of consistency of style, may be better if we use curly braces, even for the single statement.